sessions/index.js

```diff
@@ -0,0 +1,36 @@
+const {Firestore} = require('@google-cloud/firestore');
+const express = require('express');
+const session = require('express-session');
+const app = express();
+
+const {FirestoreStore} = require('@google-cloud/connect-firestore');
```

Comment:
Meta question - why put the sample here, and not in the https://github.com/googleapis/nodejs-firestore-session/ repo?

Reply:
Because it's part of the `Getting-Started` experience rather than `firestore-connect. We're  showing how to use multiple products together.

Reply:
Going to be honest, thought I was looking at the nodejs-docs-samples repo when I wrote that 🙄